### DISCUSSION TITLE
gocq v1.0.0 beta5似乎会导致cqps无法接收到消息

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#234] [一般讨论]
昨晚更新beta5之后  凌晨12点左右出现了一次  刚刚12点又出现了一次
gocq没有重连  cqps日志(normal和error)也什么都没有  单纯的机器人没有反应  重启cqps就能解决
@Tsuk1ko @sam01101 不知道是不是gocq的锅

**Comment by Tsuk1ko** [comment]
还没试过 beta5，我用一段时间看看

**Comment by Miuzarte** [comment]
很怪 昨晚没出现  今天中午又出现了
![image](https://user-images.githubusercontent.com/66856838/128456166-5dc9b8c3-7640-4429-85c5-4d05899443b8.png)
